### DISCUSSION TITLE
Meshcentral container not starting or remaining inaccessible

### DISCUSSION THREAD

**Discussion by mattague** [tteck/Proxmox#436] [Help Needed] [answered]
I'm not sure if it is because something isn't setup right but I went back through twice and used defualt settings both times. The container starts, but the script pauses at "Updating Container OS..."  

```
✓ Using local for Template Storage.
✓ Using Vault for Container Storage.
✓ Updated LXC Template List
✓ LXC Container 101 was successfully created.
✓ Started LXC Container
✓ Set up Container OS
✓ Network Connected: 10.0.0.142 
- Updating Container OS...
```

**Comment by tteck** [comment]
If it doesn't give an error, it's still updating (if the internet is reachable)
```
    __  ___          __    ______           __             __
   /  |/  /__  _____/ /_  / ____/__  ____  / /__________ _/ /
  / /|_/ / _ \/ ___/ __ \/ /   / _ \/ __ \/ __/ ___/ __  / / 
 / /  / /  __(__  ) / / / /___/  __/ / / / /_/ /  / /_/ / /  
/_/  /_/\___/____/_/ /_/\____/\___/_/ /_/\__/_/ v4\__,_/_/   

Using Advanced Settings
Using Debian Version: 11
Using Container Type: 1
Using Root Password: password
Using Container ID: 100
Using Hostname: meshcentral
Using Disk Size: 2
Allocated Cores: 1
Allocated RAM: 512
Using Bridge: vmbr0
Using IP Address: 192.168.86.252/24
Using Gateway IP Address: 192.168.86.1
Using MAC Address: Default
Using Vlan: Default
Creating a MeshCentral LXC using the above advanced settings
 ✓ Using local for Template Storage.
 ✓ Using local-lvm for Container Storage.
 ✓ Updated LXC Template List
 ✓ LXC Container 100 was successfully created.
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.252 
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Installed MeshCentral
 ✓ Cleaned
 ✓ Completed Successfully!

MeshCentral should be reachable by going to the following URL.
         http://192.168.86.252 
```

**Comment by mattague** [comment]
You reply so fast, thank you. I felt like I waited long enough, but I'm guessing not haha.

**Comment by mattague** [comment]
Ok, not sure if I broke something, but 3 hours later it still says Updating Container OS. I'm using the dev script you linked, and it shows internet is connected. I used all the default settings. Proxmox is reporting 5% CPU usage and 50% ram usage overall including other containers and virtual machines.

**Reply by tteck** [reply]
Yeah, something isn't right. I added a DNS check to the dev script as a test
```
 ✓ Started LXC Container
 ✓ Set up Container OS
 ✓ Network Connected: 192.168.86.252 
 ✓ Internet Connected
 ✓ google.com resolved to 142.250.191.142 google.com 2607
 ✓ Updated Container OS
 ✓ Installed Dependencies
 ✓ Set up Node.js Repository
 ✓ Installed Node.js
 ✓ Installed MeshCentral
 ✓ Customized Container
 ✓ Cleaned
 ✓ Completed Successfully!
```

**Comment by mattague** [comment, marked as answer]
After leaving it running for 20 minutes or so, its showing zero cpu usage. Something feels off.

![image](https://user-images.githubusercontent.com/11689183/185506229-2067908d-455d-4933-bb70-c613f6baeb7e.png)
![image](https://user-images.githubusercontent.com/11689183/185506290-a166a26a-03f6-4c69-976b-cb558429ee4c.png)

**Reply by tteck** [reply]
Is it just MeshCentral, or is it all LXC scripts doing it?

**Reply by mattague** [reply]
Tested grafana since I was planning to set that up anyway, and that worked just fine.

**Reply by tteck** [reply]
Now try the MeshCentral LXC script from the main branch. Updating Container OS is the same on both 🤷🏻‍♂️

**Reply by mattague** [reply]
Idk what you fixed, but she works.

**Reply by tteck** [reply]
lol... haven't touched the main branch `meshcentral-install.sh` since Jul 20
